src/main/java/org/team1540/robot2024/subsystems/elevator/ElevatorIO.java

```diff
@@ -0,0 +1,23 @@
+package org.team1540.robot2024.subsystems.elevator;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ElevatorIO {
+    @AutoLog
+    class ElevatorIOInputs {
+        public double positionMeters = 0.0;
+        public double velocityMPS = 0.0;
+        public double voltage = 0.0;
+        public double[] current = new double[] {};
+        public boolean atUpperLimit = false;
+        public boolean atLowerLimit = false;
+    }
+
+    default void updateInputs(ElevatorIOInputs inputs) {}
+
+    default void setPositionMeters(double position) {}
```

Comment:
this method name is somewhat misleading, because position is not something that we directly control. `setVoltage` is actually setting the applied voltage on the motor, whereas `setPositionMeters` is changing the PID setpoint, not the position. I know it looks kind of ugly but `setSetpointMeters` would be a more appropriate method name